hack/lib/common-var-init.sh

```diff
@@ -215,7 +215,19 @@ REUSE_CERTS=${REUSE_CERTS:-false}
 #
 # Avoid to create loop error at https://coredns.io/plugins/loop/#troubleshooting
 # when create coredns pod
-RESOLV_CONF=${RESOLV_CONF:-"/run/systemd/resolve/resolv.conf"}
+RESOLV_CONF=${RESOLV_CONF:-}
+if [ -z ${RESOLV_CONF} ] && [ -z ${DISABLE_NETWORK_SERVICE_SUPPORT} ]
+then
+  if [ -f "/run/systemd/resolve/resolv.conf" ]; then
+    RESOLV_CONF=${RESOLV_CONF:-"/run/systemd/resolve/resolv.conf"}
+  else
+    RESOLV_CONF=$(mktemp -p /tmp)
+    cat << 'EOF' > ${RESOLV_CONF}
+nameserver 8.8.8.8
```

Comment:
What does "nameserver 8.8.8.8" do?

BTW, I have tested this change on my host. W or w/o DISABLE_NETWORK_SERVICE_SUPPORT, all works fine. Thank you for prompt fix.

Reply:
8.8.8.8 is a well-known google public DNS service.

Reply:
How does it work in our cluster?

Reply:
hard-code 8.8.8.8 is indeed not a good idea. will change it to env var to ask user to provide a suitable DNS ip addr instead.